packages/components/src/DataList/DataList.types.ts

```diff
@@ -53,6 +53,7 @@ export interface DataListSorting extends SortableOptions {
 export interface SortableOptions {
   readonly label: string;
   readonly order: "asc" | "desc";
+  readonly dataValue: string;
```

Comment:
i wonder if this should also just be called key... or id

Reply:
I'm down to call it `id`. IMO it we shouldn't call it `key` since we already have a `key` in the same object

<img width="365" alt="Screenshot 2024-01-26 at 5 12 05 PM" src="https://github.com/GetJobber/atlantis/assets/104797704/7c611bf3-c623-4298-916b-d3175509ce4a">

Reply:
id it is!